### DISCUSSION TITLE
"Shrink to fit" the content of the columns in a grid

### DISCUSSION THREAD

**Discussion by gchumillas** [tailwindlabs/tailwindcss#4397] [Help] [answered]
It is possible make the columns of a grid "shrink to fit" the content of the columns. Simply use the "auto" value:

```html
<!-- A 2x2 table -->
<div class="inline-grid" style="grid-template-columns: auto auto">
  <!-- first row -->
  <div>Small label</div>
  <div>2</div>
  <!-- second row -->
  <div>Laaaarge label</div>
  <div>4</div>
</div>
```

But how can I translate `grid-template-columns: auto auto` to Tailwindcss classes? I don't want to use the "style" attribute.

**Comment by olets** [comment, marked as answer]
You can add your own additional `grid-template-columns` classes by configuring `gridTemplateColumns`.

https://tailwindcss.com/docs/grid-template-columns#customizing

Demo: https://play.tailwindcss.com/Y7xQ1Olpj8 (see the HTML and Config tabs)
